src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -569,17 +638,21 @@ private void SetupAnalysisInformation(Mock<ISonarQubeService> sonarQubeService,
                     new SonarQubeProjectBranch(Guid.NewGuid().ToString(), false, DateTimeOffset.MinValue)
                 };
 
-            sonarQubeService.Setup(x => x.GetProjectBranchesAsync(SharedProjectKey, CancellationToken.None))
+            sonarQubeService.Setup(x => x.GetProjectBranchesAsync(projectKey, CancellationToken.None))
                 .ReturnsAsync(projectBranches);
         }
 
         private void SetupTaintIssues(Mock<ISonarQubeService> sonarQubeService, params SonarQubeIssue[] issues)
-            => SetupTaintIssues(sonarQubeService, null, issues);
+        {
+            sonarQubeService
+                .Setup(x => x.GetTaintVulnerabilitiesAsync(It.IsAny<string>(), It.IsAny<string>(), CancellationToken.None))
```

Comment:
This overload is used when the test doesn't care what the project/branch parameters are...